gadgets/src/binary_number.rs

```diff
@@ -104,10 +148,11 @@ where
     /// Annotates columns of this gadget embedded within a circuit region.
     pub fn annotate_columns_in_region<F: Field>(&self, region: &mut Region<F>, prefix: &str) {
         let mut annotations = Vec::new();
-        for (i, _) in self.bits.iter().enumerate() {
+        for (i, _) in self.bits.0.iter().enumerate() {
             annotations.push(format!("GADGETS_binary_number_{}", i));
         }
         self.bits
+            .0
```

Comment:
We can implement a Deref for BinaryNumberBits to prevent all the `.0` diff.

Reply:
Thanks!  This was a great suggestion!  I've applied it and with it reduced the diff :D